src/components/react/ChessStudy.tsx

```diff
@@ -107,56 +104,62 @@ export const ChessStudy = ({
 					return draft;
 				}
 				case 'REMOVE_LAST_MOVE_FROM_HISTORY': {
-					if (!chessView || !draft || draft.study.moves.length == 0) return draft;
+					if (!chessView || hasNoMoves) return draft;
 
-					let moves = draft.study.moves;
+					const moves = draft.study.moves;
```

Comment:
`const` shouldn't be used for variables that are going to be modified. The use of `const` indicates that you want the variable to be treated as a constant.

Reply:
I use let and const to inform about re-assignability, not mutability and would like to keep it that way.